rep-2001.rst

```diff
@@ -33,14 +33,20 @@ Specification
 End-user entry points
 ---------------------
 
-We define three main entry points for ROS users.
-
- * desktop (recommended): The `desktop` metapackage is the main entry point for users.
-   It provides all commonly used libraries as well as visualization tools and tutorials.
- * ros_base: The `ros_base` metapackage composes the `ros_core` metapackage with commonly used libraries.
+We define various entry points for ROS users.
+
+ * `desktop_full` (recommended from Humble): The `desktop_full` variant is the main entry point for new users.
+   It provides a "batteries included" experience, enabling novice users to complete most entry
+   tutorials without knowledge of the underlying library structure.
+ * `desktop` (recommended until Galactic): The `desktop` variant provides all commonly used libraries as well as
```

Comment:
So given what we discussed earlier, I think we should change the wording here, and in particular don't say that `desktop_full` is recommended from Humble.  That's because we aren't guaranteeing that we are going to keep it available; if one of the packages in it doesn't work, we can proceed anyway.  So I'd change this to:

```suggestion
 * `desktop_full` (available from Humble): The `desktop_full` variant provides a "batteries included" experience, enabling novice users to complete most entry tutorials without knowledge of the underlying library structure.
 * `desktop` (recommended): The `desktop` variant provides all commonly used libraries as well as
```

Reply:
Fair enough, commited